Name Interests and Skills
=========================


Please fill in as appropriate.


* Shane Sturrock - C/Java/Bash etc. Linux system administration (BioIT superuser), interested in coding, performance tuning, automated testing, correctness (fit for purpose methods)

* Ruy:
  - *interests:* Fastq based read quality trim. Effect of trimming/read dropping on tag count per sample. UNEAK. Standard downstream analysis (after vcf table), sanity checks. Evaluation of sequence depth per sample. rtGBS, metagenomic GBS, exome GBS.
  - *skills:* pipeline development, Ruby, testing/debugging.

* Marcus Davy Bash/Perl/R. Quality control, Random Tag GBS, simulation validation, vcf calling workflows freebayes/samtools, Reproducible research

* Mauro Truglio: Python, R. Quality control, GBS adapters balancing, coding in general.

* Rob Elshire; Bash / AWK / general unixy stuff, GBS best practices, automation, reproducible research, TASSEL

* Patrick Biggs:
    interests:  data QC, data visualisation, comparative genomics
    skills:     Perl, MySQL, (want to have better R skills)

* Rudi:
  - *interests:* communication and feedback loops between lab staff, bioinformatician, statistician and IT staff; data visualization
  - *skills:*    Python, bash, awk, SNP chip design

* Kevin Murray:
  - *interests*: Comprehensive QC of reads and samples, simulation & validation
    tools, NextRAD, moving to reproducible pipelines, moving to using stacks
    for *de novo* GBS.
  - *skills*: C/C++, Python, R, workflow & pipeline management, sysadmin. Have
    implemented full Fastq read quality control pipeline for GBS.
  - *requirements*: Scalable and reproducible analysis pipelines, accurate and
    trustworthy *de novo* GBS analysis.
